src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/Azure.Configuration.Tests.csproj

```diff
@@ -12,7 +12,22 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Azure.Configuration\Azure.Configuration.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\..\Azure.Base\data-plane\Azure.Base.Tests\Testing\TestPool.cs" Link="TestPool.cs" />
+  </ItemGroup>
+
+  <!-- Project references -->
+  <ItemGroup Condition="'$(UseProjectReferenceToBase)'=='true'">
     <ProjectReference Include="..\..\..\Azure.Base\data-plane\Azure.Base\Azure.Base.csproj" />
-    <ProjectReference Include="..\..\..\Azure.Base\data-plane\Azure.Base.Tests\Azure.Base.Tests.csproj" />
   </ItemGroup>
+
+  <!-- Package references -->
+  <ItemGroup Condition="'$(UseProjectReferenceToBase)'==''">
```

Comment:
I would change this condition to be `'$(UseProjectReferenceToBase)'!='true'` to make sure it is always mutually exclusive to the other one.

Reply:
Done